tests/model_serving/model_server/kserve/negative/test_invalid_inference_requests.py

```
@@ -0,0 +1,141 @@
+    """
+
+    VALID_INFERENCE_BODY: dict[str, Any] = {
+        "inputs": [
+            {
+                "name": "Input3",
+                "shape": [1, 1, 28, 28],
+                "datatype": "FP32",
+                "data": [0.0] * 784,
+            }
+        ]
+    }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)